From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Fri, 9 Sep 2016 11:12:02 +0200
Subject: Fixing some spelling issues

Lintian has found some spelling issues which are not pushed upstream for
release 4.x.
---
 Documentation/changelogs/CHANGELOG-2008.txt        |  2 +-
 Documentation/changelogs/CHANGELOG-2009.txt        |  2 +-
 common/draw_panel.cpp                              |  2 +-
 common/fpid.cpp                                    |  2 +-
 common/kiway_player.cpp                            |  2 +-
 common/page_layout/page_layout_reader.cpp          |  2 +-
 common/pgm_base.cpp                                |  2 +-
 .../eeschema_component_library_editor.adoc         |  2 +-
 eeschema/dialogs/dialog_eeschema_config_fbp.cpp    |  2 +-
 eeschema/dialogs/dialog_eeschema_config_fbp.fbp    |  2 +-
 eeschema/dialogs/dialog_print_using_printer.cpp    |  4 ++--
 eeschema/edit_label.cpp                            |  2 +-
 eeschema/erc.cpp                                   |  4 ++--
 eeschema/erc.h                                     |  4 ++--
 eeschema/pinedit.cpp                               |  2 +-
 eeschema/sch_base_frame.h                          |  2 +-
 eeschema/sch_screen.cpp                            |  6 ++---
 eeschema/schframe.h                                |  4 ++--
 eeschema/viewlib_frame.cpp                         |  2 +-
 gerbview/gerbview_frame.h                          |  2 +-
 include/class_bitmap_base.h                        |  2 +-
 include/class_drawpanel.h                          |  2 +-
 include/class_undoredo_container.h                 |  2 +-
 include/dsnlexer.h                                 |  2 +-
 include/gal/opengl/glm/ext.hpp                     |  2 +-
 include/gal/opengl/glm/gtc/matrix_transform.hpp    | 26 +++++++++++-----------
 include/gal/opengl/glm/gtc/random.hpp              |  2 +-
 pagelayout_editor/pl_editor_frame.h                |  2 +-
 pcbnew/class_board.cpp                             |  2 +-
 pcbnew/connect.cpp                                 |  4 ++--
 pcbnew/connect.h                                   |  2 +-
 pcbnew/dialogs/dialog_copper_zones_base.cpp        |  2 +-
 pcbnew/dialogs/dialog_copper_zones_base.fbp        |  2 +-
 pcbnew/dialogs/dialog_design_rules.cpp             |  2 +-
 pcbnew/dialogs/dialog_export_vrml.cpp              |  2 +-
 pcbnew/dialogs/dialog_freeroute_exchange_help.html |  2 +-
 pcbnew/dialogs/dialog_gendrill_base.cpp            |  6 ++---
 pcbnew/dialogs/dialog_gendrill_base.fbp            |  6 ++---
 .../dialog_keepout_area_properties_base.cpp        |  2 +-
 .../dialog_keepout_area_properties_base.fbp        |  2 +-
 .../dialog_non_copper_zones_properties_base.cpp    |  2 +-
 .../dialog_non_copper_zones_properties_base.fbp    |  2 +-
 pcbnew/dialogs/wizard_3DShape_Libs_downloader.cpp  |  2 +-
 pcbnew/drc_clearance_test_functions.cpp            |  4 ++--
 pcbnew/onrightclick.cpp                            |  2 +-
 pcbnew/pcbnew.cpp                                  |  2 +-
 pcbnew/plot_board_layers.cpp                       |  2 +-
 pcbnew/ratsnest.cpp                                |  4 ++--
 pcbnew/zones_by_polygon.cpp                        |  6 ++---
 pcbnew/zones_functions_for_undo_redo.cpp           |  4 ++--
 pcbnew/zones_test_and_combine_areas.cpp            |  2 +-
 51 files changed, 79 insertions(+), 79 deletions(-)

diff --git a/Documentation/changelogs/CHANGELOG-2008.txt b/Documentation/changelogs/CHANGELOG-2008.txt
index d210585..c19727d 100644
--- a/Documentation/changelogs/CHANGELOG-2008.txt
+++ b/Documentation/changelogs/CHANGELOG-2008.txt
@@ -1128,7 +1128,7 @@ removed: (common/infospgm.cpp) is no longer needed
     When a sheet is deleted by block delete. it is put in undo list.
     It is really deleted after 10 changes (when the undo list is full).
     At this time, the associated data is also deleted.
-    If an other (not deleted) sheet shares the same data, this data is lost.
+    If another (not deleted) sheet shares the same data, this data is lost.
     Need work to solve this problem and keep the undo/redo feature.
 
 
diff --git a/Documentation/changelogs/CHANGELOG-2009.txt b/Documentation/changelogs/CHANGELOG-2009.txt
index fbdae4f..d58d79f 100644
--- a/Documentation/changelogs/CHANGELOG-2009.txt
+++ b/Documentation/changelogs/CHANGELOG-2009.txt
@@ -281,7 +281,7 @@ Component library object and other minor improvements.
     Speed up ZONE_CONTAINER::Copy method
     Speed up NETINFO_ITEM* BOARD::FindNet( const wxString& aNetname ) by using afast binary search
     Switch to kbool V2.1. This version solves others problems in zones calculations.
-    Adding an other way to generate thermal shape; DO NOT USE: this is for kbool tests only
+    Adding another way to generate thermal shape; DO NOT USE: this is for kbool tests only
     (I am thinking some bugs are remaining, so i am always working on kbool tests.
     Klaas Olwerda is supporting me)
 
diff --git a/common/draw_panel.cpp b/common/draw_panel.cpp
index 3bd1103..98fd282 100644
--- a/common/draw_panel.cpp
+++ b/common/draw_panel.cpp
@@ -124,7 +124,7 @@ EDA_DRAW_PANEL::EDA_DRAW_PANEL( EDA_DRAW_FRAME* parent, int id,
     m_ignoreMouseEvents = false;
     // Be sure a mouse release button event will be ignored when creating the canvas
     // if the mouse click was not made inside the canvas (can happen sometimes, when
-    // launching an editor from a double click made in an other frame)
+    // launching an editor from a double click made in another frame)
     m_ignoreNextLeftButtonRelease = true;
 
     m_mouseCaptureCallback = NULL;
diff --git a/common/fpid.cpp b/common/fpid.cpp
index ba70c67..c410bbd 100644
--- a/common/fpid.cpp
+++ b/common/fpid.cpp
@@ -167,7 +167,7 @@ int FPID::Parse( const UTF8& aId )
 
     // Be sure the footprint name is valid.
     // Some chars can be found in board file (in old board files
-    // or converted files from an other EDA tool
+    // or converted files from another EDA tool
     std::string fpname = aId.substr( partNdx, revNdx-partNdx );
     ReplaceIllegalFileNameChars( &fpname, '_' );
     SetFootprintName( UTF8( fpname ) );
diff --git a/common/kiway_player.cpp b/common/kiway_player.cpp
index a33acc1..d6fc5ae 100644
--- a/common/kiway_player.cpp
+++ b/common/kiway_player.cpp
@@ -111,7 +111,7 @@ bool KIWAY_PLAYER::ShowModal( wxString* aResult, wxWindow* aResultantFocusWindow
         // We do not want to disable top level windows which are child of the modal one,
         // if they are enabled.
         // An example is an aui toolbar which was moved
-        // or a dialog or an other frame or miniframe opened by the modal one.
+        // or a dialog or another frame or miniframe opened by the modal one.
         wxWindowList wlist = GetChildren();
         std::vector<wxWindow*> enabledTopLevelWindows;
 
diff --git a/common/page_layout/page_layout_reader.cpp b/common/page_layout/page_layout_reader.cpp
index a282989..82343b2 100644
--- a/common/page_layout/page_layout_reader.cpp
+++ b/common/page_layout/page_layout_reader.cpp
@@ -488,7 +488,7 @@ void PAGE_LAYOUT_READER_PARSER::parseGraphic( WORKSHEET_DATAITEM * aItem )
             token = NextTok();
         else
         {
-            // If an other token than T_LEFT is read here, this is an error
+            // If another token than T_LEFT is read here, this is an error
             // however, due to a old bug in kicad, the token T_end can be found
             // without T_LEFT in a very few .wks files (perhaps only one in a demo).
             // So this ugly hack disables the error detection.
diff --git a/common/pgm_base.cpp b/common/pgm_base.cpp
index 0728633..e489438 100644
--- a/common/pgm_base.cpp
+++ b/common/pgm_base.cpp
@@ -669,7 +669,7 @@ bool PGM_BASE::SetLanguage( bool first_time )
 
     if( !first_time )
     {
-        // If we are here, the user has selected an other language.
+        // If we are here, the user has selected another language.
         // Therefore the new prefered language name is stored in common config.
         // Do NOT store the wxWidgets language Id, it can change between wxWidgets
         // versions, for a given language
diff --git a/doc/src/eeschema/eeschema_component_library_editor.adoc b/doc/src/eeschema/eeschema_component_library_editor.adoc
index 29312a9..4b76668 100644
--- a/doc/src/eeschema/eeschema_component_library_editor.adoc
+++ b/doc/src/eeschema/eeschema_component_library_editor.adoc
@@ -406,7 +406,7 @@ existing component.
   image:images/icons/save_part_in_mem.png[icons/save_part_in_mem_png]
   or save to a new library by clicking the 
   image:images/icons/new_library.png[icons/new_library_png]
-  or if you want to save this new component in an other existing
+  or if you want to save this new component in another existing
   library select the other library by clicking on the
   image:images/icons/library.png[icons/library_png]
   and save the new component.
diff --git a/eeschema/dialogs/dialog_eeschema_config_fbp.cpp b/eeschema/dialogs/dialog_eeschema_config_fbp.cpp
index 5ac72bc..c803aa5 100644
--- a/eeschema/dialogs/dialog_eeschema_config_fbp.cpp
+++ b/eeschema/dialogs/dialog_eeschema_config_fbp.cpp
@@ -27,7 +27,7 @@ DIALOG_EESCHEMA_CONFIG_FBP::DIALOG_EESCHEMA_CONFIG_FBP( wxWindow* parent, wxWind
 	bSizerLibsChoice = new wxBoxSizer( wxHORIZONTAL );
 	
 	m_ListLibr = new wxListBox( this, wxID_ANY, wxDefaultPosition, wxDefaultSize, 0, NULL, wxLB_EXTENDED|wxLB_HSCROLL|wxLB_NEEDED_SB|wxLB_SINGLE ); 
-	m_ListLibr->SetToolTip( _("List of active library files.\nOnly library files in this list are loaded by Eeschema.\nThe order of this list is important:\nEeschema searchs for a given component using this list order priority.") );
+	m_ListLibr->SetToolTip( _("List of active library files.\nOnly library files in this list are loaded by Eeschema.\nThe order of this list is important:\nEeschema searches for a given component using this list order priority.") );
 	m_ListLibr->SetMinSize( wxSize( 400,250 ) );
 	
 	bSizerLibsChoice->Add( m_ListLibr, 1, wxEXPAND|wxBOTTOM|wxRIGHT|wxLEFT, 5 );
diff --git a/eeschema/dialogs/dialog_eeschema_config_fbp.fbp b/eeschema/dialogs/dialog_eeschema_config_fbp.fbp
index 72fcc43..2abdd17 100644
--- a/eeschema/dialogs/dialog_eeschema_config_fbp.fbp
+++ b/eeschema/dialogs/dialog_eeschema_config_fbp.fbp
@@ -247,7 +247,7 @@
                                         <property name="style">wxLB_EXTENDED|wxLB_HSCROLL|wxLB_NEEDED_SB|wxLB_SINGLE</property>
                                         <property name="subclass"></property>
                                         <property name="toolbar_pane">0</property>
-                                        <property name="tooltip">List of active library files.&#x0A;Only library files in this list are loaded by Eeschema.&#x0A;The order of this list is important:&#x0A;Eeschema searchs for a given component using this list order priority.</property>
+                                        <property name="tooltip">List of active library files.&#x0A;Only library files in this list are loaded by Eeschema.&#x0A;The order of this list is important:&#x0A;Eeschema searches for a given component using this list order priority.</property>
                                         <property name="validator_data_type"></property>
                                         <property name="validator_style">wxFILTER_NONE</property>
                                         <property name="validator_type">wxDefaultValidator</property>
diff --git a/eeschema/dialogs/dialog_print_using_printer.cpp b/eeschema/dialogs/dialog_print_using_printer.cpp
index 09257bd..874a6d4 100644
--- a/eeschema/dialogs/dialog_print_using_printer.cpp
+++ b/eeschema/dialogs/dialog_print_using_printer.cpp
@@ -272,7 +272,7 @@ void DIALOG_PRINT_USING_PRINTER::OnPrintPreview( wxCommandEvent& event )
 
     // We use here wxPreviewFrame_WindowModal option to make the wxPrintPreview frame
     // modal for its caller only.
-    // An other reason is the fact when closing the frame without this option,
+    // another reason is the fact when closing the frame without this option,
     // all top level frames are reenabled.
     // With this option, only the parent is reenabled.
     // Reenabling all top level frames should be made by the parent dialog.
@@ -439,7 +439,7 @@ void SCH_PRINTOUT::DrawPage( SCH_SCREEN* aScreen )
 
     // For an obscure reason, OffsetLogicalOrigin creates issues,
     // under some circumstances, when yoffset is not always null
-    // and changes from a page to an other page
+    // and changes from a page to another page
     // This is only a workaround, not a fix
     // see https://bugs.launchpad.net/kicad/+bug/1464773
     // xoffset does not create issues.
diff --git a/eeschema/edit_label.cpp b/eeschema/edit_label.cpp
index 2be1d87..060db3b 100644
--- a/eeschema/edit_label.cpp
+++ b/eeschema/edit_label.cpp
@@ -131,7 +131,7 @@ SCH_TEXT* SCH_EDIT_FRAME::CreateNewText( wxDC* aDC, int aType )
 
 
 /*
- * OnConvertTextType is a command event handler to change a text type to an other one.
+ * OnConvertTextType is a command event handler to change a text type to another one.
  * The new text, label, hierarchical label, or global label is created from the old text
  * The old text is deleted.
  * A tricky case is when the 'old" text is being edited (i.e. moving)
diff --git a/eeschema/erc.cpp b/eeschema/erc.cpp
index 099249c..0ae9c5c 100644
--- a/eeschema/erc.cpp
+++ b/eeschema/erc.cpp
@@ -405,7 +405,7 @@ void TestOthersItems( NETLIST_OBJECT_LIST* aList,
                 {
                     /* This pin is not connected: for multiple part per
                      * package, and duplicated pin,
-                     * search for an other instance of this pin
+                     * search for another instance of this pin
                      * this will be flagged only if all instances of this pin
                      * are not connected
                      * TODO test also if instances connected are connected to
@@ -430,7 +430,7 @@ void TestOthersItems( NETLIST_OBJECT_LIST* aList,
                             continue;
 
                         // Same component and same pin. Do dot create error for this pin
-                        // if the other pin is connected (i.e. if duplicate net has an other
+                        // if the other pin is connected (i.e. if duplicate net has another
                         // item)
                         if( (duplicate > 0)
                           && ( aList->GetItemNet( duplicate ) ==
diff --git a/eeschema/erc.h b/eeschema/erc.h
index 708fb6a..8654b48 100644
--- a/eeschema/erc.h
+++ b/eeschema/erc.h
@@ -53,8 +53,8 @@ extern const wxString CommentERC_V[];
 #define ERCE_DUPLICATE_SHEET_NAME 1    // duplicate sheet names within a given sheet
 #define ERCE_PIN_NOT_CONNECTED    2    // pin not connected and not no connect symbol
 #define ERCE_PIN_NOT_DRIVEN       3    // pin connected to some others pins but no pin to drive it
-#define ERCE_PIN_TO_PIN_WARNING   4    // pin connected to an other pin: warning level
-#define ERCE_PIN_TO_PIN_ERROR     5    // pin connected to an other pin: error level
+#define ERCE_PIN_TO_PIN_WARNING   4    // pin connected to another pin: warning level
+#define ERCE_PIN_TO_PIN_ERROR     5    // pin connected to another pin: error level
 #define ERCE_HIERACHICAL_LABEL    6    // mismatch between hierarchical labels and pins sheets
 #define ERCE_NOCONNECT_CONNECTED  7    // a no connect symbol is connected to more than 1 pin
 #define ERCE_GLOBLABEL            8    // global label not connected to any other global label
diff --git a/eeschema/pinedit.cpp b/eeschema/pinedit.cpp
index 1b44172..96a9c10 100644
--- a/eeschema/pinedit.cpp
+++ b/eeschema/pinedit.cpp
@@ -236,7 +236,7 @@ void LIB_EDIT_FRAME::PlacePin()
 
     LIB_PART*      part = GetCurPart();
 
-    // Test for an other pin in same new position:
+    // Test for another pin in same new position:
     for( LIB_PIN* pin = part->GetNextPin();  pin;  pin = part->GetNextPin( pin ) )
     {
         if( pin == cur_pin || newpos != pin->GetPosition() || pin->GetFlags() )
diff --git a/eeschema/sch_base_frame.h b/eeschema/sch_base_frame.h
index c99e1a6..1ff78e3 100644
--- a/eeschema/sch_base_frame.h
+++ b/eeschema/sch_base_frame.h
@@ -169,7 +169,7 @@ protected:
      * Function OnOpenLibraryViewer
      * Open the library viewer only to browse library contents.
      * If the viewed is already opened from this, raise the viewer
-     * If the viewed is already opened from an other window, close it and reopen
+     * If the viewed is already opened from another window, close it and reopen
      */
     void OnOpenLibraryViewer( wxCommandEvent& event );
 
diff --git a/eeschema/sch_screen.cpp b/eeschema/sch_screen.cpp
index c2f8201..b6f052e 100644
--- a/eeschema/sch_screen.cpp
+++ b/eeschema/sch_screen.cpp
@@ -825,7 +825,7 @@ void SCH_SCREEN::SelectBlockItems()
                 // this selected wire has no ends in block.
                 // But it was selected (because it intersects the selecting area),
                 // so we must keep it selected and select items connected to it
-                // Note: an other option could be: remove it from drag list
+                // Note: another option could be: remove it from drag list
                 item->SetFlags( SELECTED | SKIP_STRUCT );
                 std::vector< wxPoint > connections;
                 item->GetConnectionPoints( connections );
@@ -1227,7 +1227,7 @@ int SCH_SCREEN::GetConnection( const wxPoint& aPosition, PICKED_ITEMS_LIST& aLis
 
             // when tmp != NULL, segment is a new candidate:
             // put it in deleted list if
-            // the start point is not connected to an other item (like pin)
+            // the start point is not connected to another item (like pin)
             if( tmp && !CountConnectedItems( segment->GetStartPoint(), true ) )
                 noconnect = true;
 
@@ -1251,7 +1251,7 @@ int SCH_SCREEN::GetConnection( const wxPoint& aPosition, PICKED_ITEMS_LIST& aLis
 
             // when tmp != NULL, segment is a new candidate:
             // put it in deleted list if
-            // the end point is not connected to an other item (like pin)
+            // the end point is not connected to another item (like pin)
             if( tmp && !CountConnectedItems( segment->GetEndPoint(), true ) )
                 noconnect = true;
 
diff --git a/eeschema/schframe.h b/eeschema/schframe.h
index 0db8ea8..693331a 100644
--- a/eeschema/schframe.h
+++ b/eeschema/schframe.h
@@ -903,7 +903,7 @@ private:
 
     /**
      * Function OnCovertTextType
-     * is a command event handler to change a text type to an other one.  The new text,
+     * is a command event handler to change a text type to another one.  The new text,
      * label, hierarchical label, or global label is created from the old text and the
      * old text is deleted.
      */
@@ -913,7 +913,7 @@ private:
      * Function BeginSegment
      * creates a new segment ( WIRE, BUS ) or terminates the current segment in progress.
      *
-     * If the end of the current segment is on an other segment, place a junction if needed
+     * If the end of the current segment is on another segment, place a junction if needed
      * and terminates the command.  If the end of the current segment is on a pin, terminate
      * the command.  In all other cases starts a new segment.
      */
diff --git a/eeschema/viewlib_frame.cpp b/eeschema/viewlib_frame.cpp
index ddcefa9..ed13bfa 100644
--- a/eeschema/viewlib_frame.cpp
+++ b/eeschema/viewlib_frame.cpp
@@ -443,7 +443,7 @@ void LIB_VIEW_FRAME::SetSelectedLibrary( const wxString& aLibraryName )
 
     // Ensure the corresponding line in m_libList is selected
     // (which is not necessary the case if SetSelectedLibrary is called
-    // by an other caller than ClickOnLibList.
+    // by another caller than ClickOnLibList.
     m_libList->SetStringSelection( m_libraryName, true );
 }
 
diff --git a/gerbview/gerbview_frame.h b/gerbview/gerbview_frame.h
index 383ab54..4675b2b 100644
--- a/gerbview/gerbview_frame.h
+++ b/gerbview/gerbview_frame.h
@@ -355,7 +355,7 @@ public:
     /**
      * Function GetNegativeItemsColor
      * @return the color of negative items.
-     * This is usually the background color, but can be an other color
+     * This is usually the background color, but can be another color
      * in order to see negative objects
      */
     EDA_COLOR_T GetNegativeItemsColor() const;
diff --git a/include/class_bitmap_base.h b/include/class_bitmap_base.h
index 53c5b83..9fe28e9 100644
--- a/include/class_bitmap_base.h
+++ b/include/class_bitmap_base.h
@@ -35,7 +35,7 @@ class PLOTTER;
 
 /**
  * This class handle bitmap images in KiCad.
- * It is not intended to be used alone, but inside an other class,
+ * It is not intended to be used alone, but inside another class,
  * so all methods are protected ( or private )
  * It is used in SCH_BITMAP class and WS_DRAW_ITEM_BITMAP (and other in future)
  *
diff --git a/include/class_drawpanel.h b/include/class_drawpanel.h
index 14ad104..5f71a04 100644
--- a/include/class_drawpanel.h
+++ b/include/class_drawpanel.h
@@ -113,7 +113,7 @@ private:
     END_MOUSE_CAPTURE_CALLBACK m_endMouseCaptureCallback;
 
     /// useful to avoid false start block in certain cases
-    /// (like switch from a sheet to an other sheet
+    /// (like switch from a sheet to another sheet
     /// >= 0 (or >= n) if a block can start
     int     m_canStartBlock;
 
diff --git a/include/class_undoredo_container.h b/include/class_undoredo_container.h
index 13c9e85..f3473db 100644
--- a/include/class_undoredo_container.h
+++ b/include/class_undoredo_container.h
@@ -89,7 +89,7 @@ private:
                                         * edited item. */
     KICAD_T        m_pickedItemType;   /* type of schematic or board item that is concerned */
 
-    EDA_ITEM*      m_link;             /* Pointer on an other item. Used in undo redo command
+    EDA_ITEM*      m_link;             /* Pointer on another item. Used in undo redo command
                                         * used when a duplicate exists i.e. when an item is
                                         * modified, and the copy of initial item exists (the
                                         * duplicate) m_Item points the duplicate (i.e the old
diff --git a/include/dsnlexer.h b/include/dsnlexer.h
index ded81c7..33bbc43 100644
--- a/include/dsnlexer.h
+++ b/include/dsnlexer.h
@@ -219,7 +219,7 @@ public:
      * Useable only for DSN lexers which share the same LINE_READER
      * Synchronizes the pointers handling the data read by the LINE_READER
      * Allows 2 DNSLEXER to share the same current line, when switching from a
-     * DNSLEXER to an other DNSLEXER
+     * DNSLEXER to another DNSLEXER
      * @param aLexer = the model
      * @return true if the sync can be made ( at least the same line reader )
      */
diff --git a/include/gal/opengl/glm/ext.hpp b/include/gal/opengl/glm/ext.hpp
index 9ef0c92..2068e43 100644
--- a/include/gal/opengl/glm/ext.hpp
+++ b/include/gal/opengl/glm/ext.hpp
@@ -60,7 +60,7 @@
 
 #if(defined(GLM_MESSAGES) && !defined(GLM_MESSAGE_EXT_INCLUDED_DISPLAYED))
 #	define GLM_MESSAGE_EXT_INCLUDED_DISPLAYED
-#	pragma message("GLM: All extensions included (not recommanded)")
+#	pragma message("GLM: All extensions included (not recommended)")
 #endif//GLM_MESSAGES
 
 #include "./gtc/constants.hpp"
diff --git a/include/gal/opengl/glm/gtc/matrix_transform.hpp b/include/gal/opengl/glm/gtc/matrix_transform.hpp
index 020c6dd..3fc55c9 100644
--- a/include/gal/opengl/glm/gtc/matrix_transform.hpp
+++ b/include/gal/opengl/glm/gtc/matrix_transform.hpp
@@ -62,7 +62,7 @@ namespace glm
 	/// 
 	/// @param m Input matrix multiplied by this translation matrix.
 	/// @param v Coordinates of a translation vector.
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @code
 	/// #include <glm/glm.hpp>
 	/// #include <glm/gtc/matrix_transform.hpp>
@@ -87,7 +87,7 @@ namespace glm
 	/// 
 	/// @param m Input matrix multiplied by this rotation matrix.
 	/// @param angle Rotation angle expressed in radians if GLM_FORCE_RADIANS is define or degrees otherwise.
-	/// @param axis Rotation axis, recommanded to be normalized.
+	/// @param axis Rotation axis, recommended to be normalized.
 	/// @tparam T Value type used to build the matrix. Supported: half, float or double.
 	/// @see gtc_matrix_transform
 	/// @see gtx_transform
@@ -104,7 +104,7 @@ namespace glm
 	/// 
 	/// @param m Input matrix multiplied by this scale matrix.
 	/// @param v Ratio of scaling for each axis.
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
 	/// @see gtx_transform
 	/// @see - scale(T x, T y, T z) scale(T const & x, T const & y, T const & z)
@@ -123,7 +123,7 @@ namespace glm
 	/// @param top 
 	/// @param zNear 
 	/// @param zFar 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
 	/// @see - glm::ortho(T const & left, T const & right, T const & bottom, T const & top)
 	template <typename T> 
@@ -141,7 +141,7 @@ namespace glm
 	/// @param right 
 	/// @param bottom 
 	/// @param top 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
     /// @see - glm::ortho(T const & left, T const & right, T const & bottom, T const & top, T const & zNear, T const & zFar)
 	template <typename T> 
@@ -159,7 +159,7 @@ namespace glm
 	/// @param top 
 	/// @param near 
 	/// @param far 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
 	template <typename T> 
 	detail::tmat4x4<T> frustum(
@@ -176,7 +176,7 @@ namespace glm
 	/// @param aspect 
 	/// @param near 
 	/// @param far 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
 	template <typename T> 
 	detail::tmat4x4<T> perspective(
@@ -192,7 +192,7 @@ namespace glm
 	/// @param height 
 	/// @param near 
 	/// @param far 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
 	template <typename valType> 
 	detail::tmat4x4<valType> perspectiveFov(
@@ -207,7 +207,7 @@ namespace glm
 	/// @param fovy Expressed in radians if GLM_FORCE_RADIANS is define or degrees otherwise.
 	/// @param aspect 
 	/// @param near 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
     template <typename T> 
 	detail::tmat4x4<T> infinitePerspective(
@@ -218,7 +218,7 @@ namespace glm
 	/// @param fovy Expressed in radians if GLM_FORCE_RADIANS is define or degrees otherwise.
 	/// @param aspect 
 	/// @param near 
-	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Value type used to build the matrix. Currently supported: half (not recommended), float or double.
 	/// @see gtc_matrix_transform
     template <typename T> 
 	detail::tmat4x4<T> tweakedInfinitePerspective(
@@ -230,7 +230,7 @@ namespace glm
 	/// @param model 
 	/// @param proj
 	/// @param viewport 
-	/// @tparam T Native type used for the computation. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Native type used for the computation. Currently supported: half (not recommended), float or double.
 	/// @tparam U Currently supported: Floating-point types and integer types.
 	/// @see gtc_matrix_transform
 	template <typename T, typename U> 
@@ -246,7 +246,7 @@ namespace glm
 	/// @param model 
 	/// @param proj
 	/// @param viewport 
-	/// @tparam T Native type used for the computation. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Native type used for the computation. Currently supported: half (not recommended), float or double.
 	/// @tparam U Currently supported: Floating-point types and integer types.
 	/// @see gtc_matrix_transform
 	template <typename T, typename U> 
@@ -261,7 +261,7 @@ namespace glm
 	/// @param center 
 	/// @param delta 
 	/// @param viewport
-	/// @tparam T Native type used for the computation. Currently supported: half (not recommanded), float or double.
+	/// @tparam T Native type used for the computation. Currently supported: half (not recommended), float or double.
 	/// @tparam U Currently supported: Floating-point types and integer types.
 	/// @see gtc_matrix_transform
 	template <typename T, typename U> 
diff --git a/include/gal/opengl/glm/gtc/random.hpp b/include/gal/opengl/glm/gtc/random.hpp
index 3cda59c..98d9b56 100644
--- a/include/gal/opengl/glm/gtc/random.hpp
+++ b/include/gal/opengl/glm/gtc/random.hpp
@@ -57,7 +57,7 @@ namespace glm
 	/// 
 	/// @param Min 
 	/// @param Max 
-	/// @tparam genType Value type. Currently supported: half (not recommanded), float or double scalars and vectors.
+	/// @tparam genType Value type. Currently supported: half (not recommended), float or double scalars and vectors.
 	/// @see gtc_random
 	template <typename genType> 
 	genType linearRand(
diff --git a/pagelayout_editor/pl_editor_frame.h b/pagelayout_editor/pl_editor_frame.h
index ba3b7ae..e278d99 100644
--- a/pagelayout_editor/pl_editor_frame.h
+++ b/pagelayout_editor/pl_editor_frame.h
@@ -177,7 +177,7 @@ public:
 
     /**
      * Page layout editor can show the title block using a page number
-     * 1 or an other number.
+     * 1 or another number.
      * This is because some items can be shown (or not) only on page 1
      * (a feature  which look like word processing option
      * "page 1 differs from other pages".
diff --git a/pcbnew/class_board.cpp b/pcbnew/class_board.cpp
index a909050..fb64b06 100644
--- a/pcbnew/class_board.cpp
+++ b/pcbnew/class_board.cpp
@@ -203,7 +203,7 @@ void BOARD::chainMarkedSegments( wxPoint aPosition, const LSET& aLayerMask, TRAC
      * end or not.  They will be analyzer later and vias on terminal points
      * of the track will be considered as part of this track if they do not
      * connect segments of another track together and will be considered as
-     * part of an other track when removing the via, the segments of that other
+     * part of another track when removing the via, the segments of that other
      * track are disconnected.
      */
     for( ; ; )
diff --git a/pcbnew/connect.cpp b/pcbnew/connect.cpp
index 848c30b..12bb6a6 100644
--- a/pcbnew/connect.cpp
+++ b/pcbnew/connect.cpp
@@ -570,7 +570,7 @@ int CONNECTIONS::Merge_SubNets( int aOldSubNet, int aNewSubNet )
  * segments connected together.
  * The track list must be sorted by nets, and all segments
  * from m_firstTrack to m_lastTrack have the same net
- * When 2 items are connected (a track to a pad, or a track to an other track),
+ * When 2 items are connected (a track to a pad, or a track to another track),
  * they are grouped in a cluster.
  * The .m_Subnet member is the cluster identifier (subnet id)
  * For a given net, if all tracks are created, there is only one cluster.
@@ -655,7 +655,7 @@ void CONNECTIONS::Propagate_SubNets()
                 }
                 else
                 {
-                    // it is connected to an other segment not in a cluster, so we must
+                    // it is connected to another segment not in a cluster, so we must
                     // create a new cluster (only with the 2 track segments)
                     sub_netcode++;
                     curr_track->SetSubNet( sub_netcode );
diff --git a/pcbnew/connect.h b/pcbnew/connect.h
index 46868c8..89cb4dc 100644
--- a/pcbnew/connect.h
+++ b/pcbnew/connect.h
@@ -213,7 +213,7 @@ public:
      * segments connected together.
      * The track list must be sorted by nets, and all segments
      * from m_firstTrack to m_lastTrack have the same net.
-     * When 2 items are connected (a track to a pad, or a track to an other track),
+     * When 2 items are connected (a track to a pad, or a track to another track),
      * they are grouped in a cluster.
      * For pads, this is the .m_physical_connexion member which is a cluster identifier
      * For tracks, this is the .m_Subnet member which is a cluster identifier
diff --git a/pcbnew/dialogs/dialog_copper_zones_base.cpp b/pcbnew/dialogs/dialog_copper_zones_base.cpp
index 75cf1a2..3735a4f 100644
--- a/pcbnew/dialogs/dialog_copper_zones_base.cpp
+++ b/pcbnew/dialogs/dialog_copper_zones_base.cpp
@@ -184,7 +184,7 @@ DIALOG_COPPER_ZONE_BASE::DIALOG_COPPER_ZONE_BASE( wxWindow* parent, wxWindowID i
 	
 	m_staticText171 = new wxStaticText( m_ExportableSetupSizer->GetStaticBox(), wxID_ANY, _("Priority level:"), wxDefaultPosition, wxDefaultSize, 0 );
 	m_staticText171->Wrap( -1 );
-	m_staticText171->SetToolTip( _("Zones are filled by priority level, level 3 has higher priority than level 2.\nWhen a zone is inside an other zone:\n* If its priority is higher, its outlines are removed from the other zone.\n* If its priority is equal, a DRC error is set.") );
+	m_staticText171->SetToolTip( _("Zones are filled by priority level, level 3 has higher priority than level 2.\nWhen a zone is inside another zone:\n* If its priority is higher, its outlines are removed from the other zone.\n* If its priority is equal, a DRC error is set.") );
 	
 	m_MiddleBox->Add( m_staticText171, 0, wxTOP|wxRIGHT|wxLEFT, 5 );
 	
diff --git a/pcbnew/dialogs/dialog_copper_zones_base.fbp b/pcbnew/dialogs/dialog_copper_zones_base.fbp
index 75d5026..700556b 100644
--- a/pcbnew/dialogs/dialog_copper_zones_base.fbp
+++ b/pcbnew/dialogs/dialog_copper_zones_base.fbp
@@ -2431,7 +2431,7 @@
                                         <property name="style"></property>
                                         <property name="subclass"></property>
                                         <property name="toolbar_pane">0</property>
-                                        <property name="tooltip">Zones are filled by priority level, level 3 has higher priority than level 2.&#x0A;When a zone is inside an other zone:&#x0A;* If its priority is higher, its outlines are removed from the other zone.&#x0A;* If its priority is equal, a DRC error is set.</property>
+                                        <property name="tooltip">Zones are filled by priority level, level 3 has higher priority than level 2.&#x0A;When a zone is inside another zone:&#x0A;* If its priority is higher, its outlines are removed from the other zone.&#x0A;* If its priority is equal, a DRC error is set.</property>
                                         <property name="window_extra_style"></property>
                                         <property name="window_name"></property>
                                         <property name="window_style"></property>
diff --git a/pcbnew/dialogs/dialog_design_rules.cpp b/pcbnew/dialogs/dialog_design_rules.cpp
index 80fe0f8..6b76262 100644
--- a/pcbnew/dialogs/dialog_design_rules.cpp
+++ b/pcbnew/dialogs/dialog_design_rules.cpp
@@ -511,7 +511,7 @@ void DIALOG_DESIGN_RULES::CopyRulesListToBoard()
 
         if( !m_BrdSettings->m_NetClasses.Add( nc ) )
         {
-            // this netclass cannot be added because an other netclass with the same name exists
+            // this netclass cannot be added because another netclass with the same name exists
             // Should not occur because OnAddNetclassClick() tests for existing NetClass names
             wxString msg;
             msg.Printf( wxT( "CopyRulesListToBoard(): The NetClass \"%s\" already exists. Skip" ),
diff --git a/pcbnew/dialogs/dialog_export_vrml.cpp b/pcbnew/dialogs/dialog_export_vrml.cpp
index c2165b9..b3af7cc 100644
--- a/pcbnew/dialogs/dialog_export_vrml.cpp
+++ b/pcbnew/dialogs/dialog_export_vrml.cpp
@@ -184,7 +184,7 @@ void PCB_EDIT_FRAME::OnExportVRML( wxCommandEvent& event )
     static wxString last_vrmlName;  // the last wrml file name built
 
     // If the board name has changed since the last export,
-    // do not use the old path, initialized by an other board
+    // do not use the old path, initialized by another board
     if( last_brdName.IsEmpty() || last_brdName != GetBoard()->GetFileName() )
     {
         last_brdName = GetBoard()->GetFileName();
diff --git a/pcbnew/dialogs/dialog_freeroute_exchange_help.html b/pcbnew/dialogs/dialog_freeroute_exchange_help.html
index 375df66..61eb6ff 100644
--- a/pcbnew/dialogs/dialog_freeroute_exchange_help.html
+++ b/pcbnew/dialogs/dialog_freeroute_exchange_help.html
@@ -40,7 +40,7 @@
 <li> in freerouter: load the project's *.dsn file. Immediately after a load, all
      components and traces (if any) will initially be 'fixed'. This is a 'lock
      in place' toggle that you can undo by selecting a region with your mouse
-     and then selecting 'Unfix' from the menu. Occassionally you may want to
+     and then selecting 'Unfix' from the menu. Occasionally you may want to
      re-fix a trace or a part, if only temporarily. This keeps it locked in
      place.
 </li><br>
diff --git a/pcbnew/dialogs/dialog_gendrill_base.cpp b/pcbnew/dialogs/dialog_gendrill_base.cpp
index e868e1f..92451b9 100644
--- a/pcbnew/dialogs/dialog_gendrill_base.cpp
+++ b/pcbnew/dialogs/dialog_gendrill_base.cpp
@@ -83,17 +83,17 @@ DIALOG_GENDRILL_BASE::DIALOG_GENDRILL_BASE( wxWindow* parent, wxWindowID id, con
 	sbOptSizer = new wxStaticBoxSizer( new wxStaticBox( this, wxID_ANY, _("Drill File Options:") ), wxVERTICAL );
 	
 	m_Check_Mirror = new wxCheckBox( sbOptSizer->GetStaticBox(), wxID_ANY, _("Mirror y axis"), wxDefaultPosition, wxDefaultSize, 0 );
-	m_Check_Mirror->SetToolTip( _("Not recommanded.\nUsed mostly by users who make themselves the boards.") );
+	m_Check_Mirror->SetToolTip( _("Not recommended.\nUsed mostly by users who make themselves the boards.") );
 	
 	sbOptSizer->Add( m_Check_Mirror, 0, wxRIGHT|wxLEFT, 5 );
 	
 	m_Check_Minimal = new wxCheckBox( sbOptSizer->GetStaticBox(), wxID_ANY, _("Minimal header"), wxDefaultPosition, wxDefaultSize, 0 );
-	m_Check_Minimal->SetToolTip( _("Not recommanded.\nUse it only for board houses which do not accept fully featured headers.") );
+	m_Check_Minimal->SetToolTip( _("Not recommended.\nUse it only for board houses which do not accept fully featured headers.") );
 	
 	sbOptSizer->Add( m_Check_Minimal, 0, wxTOP|wxRIGHT|wxLEFT, 5 );
 	
 	m_Check_Merge_PTH_NPTH = new wxCheckBox( sbOptSizer->GetStaticBox(), wxID_ANY, _("Merge PTH and NPTH holes into one file"), wxDefaultPosition, wxDefaultSize, 0 );
-	m_Check_Merge_PTH_NPTH->SetToolTip( _("Not recommanded.\nUse it only for board houses which ask for merged PTH and NPTH into onlu one file") );
+	m_Check_Merge_PTH_NPTH->SetToolTip( _("Not recommended.\nUse it only for board houses which ask for merged PTH and NPTH into onlu one file") );
 	
 	sbOptSizer->Add( m_Check_Merge_PTH_NPTH, 0, wxALL, 5 );
 	
diff --git a/pcbnew/dialogs/dialog_gendrill_base.fbp b/pcbnew/dialogs/dialog_gendrill_base.fbp
index 73fef32..8ccb9e9 100644
--- a/pcbnew/dialogs/dialog_gendrill_base.fbp
+++ b/pcbnew/dialogs/dialog_gendrill_base.fbp
@@ -759,7 +759,7 @@
                                                 <property name="style"></property>
                                                 <property name="subclass"></property>
                                                 <property name="toolbar_pane">0</property>
-                                                <property name="tooltip">Not recommanded.&#x0A;Used mostly by users who make themselves the boards.</property>
+                                                <property name="tooltip">Not recommended.&#x0A;Used mostly by users who make themselves the boards.</property>
                                                 <property name="validator_data_type"></property>
                                                 <property name="validator_style">wxFILTER_NONE</property>
                                                 <property name="validator_type">wxDefaultValidator</property>
@@ -847,7 +847,7 @@
                                                 <property name="style"></property>
                                                 <property name="subclass"></property>
                                                 <property name="toolbar_pane">0</property>
-                                                <property name="tooltip">Not recommanded.&#x0A;Use it only for board houses which do not accept fully featured headers.</property>
+                                                <property name="tooltip">Not recommended.&#x0A;Use it only for board houses which do not accept fully featured headers.</property>
                                                 <property name="validator_data_type"></property>
                                                 <property name="validator_style">wxFILTER_NONE</property>
                                                 <property name="validator_type">wxDefaultValidator</property>
@@ -935,7 +935,7 @@
                                                 <property name="style"></property>
                                                 <property name="subclass"></property>
                                                 <property name="toolbar_pane">0</property>
-                                                <property name="tooltip">Not recommanded.&#x0A;Use it only for board houses which ask for merged PTH and NPTH into onlu one file</property>
+                                                <property name="tooltip">Not recommended.&#x0A;Use it only for board houses which ask for merged PTH and NPTH into onlu one file</property>
                                                 <property name="validator_data_type"></property>
                                                 <property name="validator_style">wxFILTER_NONE</property>
                                                 <property name="validator_type">wxDefaultValidator</property>
diff --git a/pcbnew/dialogs/dialog_keepout_area_properties_base.cpp b/pcbnew/dialogs/dialog_keepout_area_properties_base.cpp
index 0665261..6a83094 100644
--- a/pcbnew/dialogs/dialog_keepout_area_properties_base.cpp
+++ b/pcbnew/dialogs/dialog_keepout_area_properties_base.cpp
@@ -52,7 +52,7 @@ DIALOG_KEEPOUT_AREA_PROPERTIES_BASE::DIALOG_KEEPOUT_AREA_PROPERTIES_BASE( wxWind
 	
 	wxString m_OutlineAppearanceCtrlChoices[] = { _("Line"), _("Hatched Outline"), _("Full Hatched") };
 	int m_OutlineAppearanceCtrlNChoices = sizeof( m_OutlineAppearanceCtrlChoices ) / sizeof( wxString );
-	m_OutlineAppearanceCtrl = new wxRadioBox( this, wxID_ANY, _("Outline Appearence:"), wxDefaultPosition, wxDefaultSize, m_OutlineAppearanceCtrlNChoices, m_OutlineAppearanceCtrlChoices, 1, wxRA_SPECIFY_COLS );
+	m_OutlineAppearanceCtrl = new wxRadioBox( this, wxID_ANY, _("Outline Appearance:"), wxDefaultPosition, wxDefaultSize, m_OutlineAppearanceCtrlNChoices, m_OutlineAppearanceCtrlChoices, 1, wxRA_SPECIFY_COLS );
 	m_OutlineAppearanceCtrl->SetSelection( 1 );
 	bSizerRight->Add( m_OutlineAppearanceCtrl, 0, wxALL|wxEXPAND, 5 );
 	
diff --git a/pcbnew/dialogs/dialog_keepout_area_properties_base.fbp b/pcbnew/dialogs/dialog_keepout_area_properties_base.fbp
index 926ae8b..5fc7b1c 100644
--- a/pcbnew/dialogs/dialog_keepout_area_properties_base.fbp
+++ b/pcbnew/dialogs/dialog_keepout_area_properties_base.fbp
@@ -516,7 +516,7 @@
                                         <property name="gripper">0</property>
                                         <property name="hidden">0</property>
                                         <property name="id">wxID_ANY</property>
-                                        <property name="label">Outline Appearence:</property>
+                                        <property name="label">Outline Appearance:</property>
                                         <property name="majorDimension">1</property>
                                         <property name="max_size"></property>
                                         <property name="maximize_button">0</property>
diff --git a/pcbnew/dialogs/dialog_non_copper_zones_properties_base.cpp b/pcbnew/dialogs/dialog_non_copper_zones_properties_base.cpp
index c41ffc2..6d32e86 100644
--- a/pcbnew/dialogs/dialog_non_copper_zones_properties_base.cpp
+++ b/pcbnew/dialogs/dialog_non_copper_zones_properties_base.cpp
@@ -51,7 +51,7 @@ DIALOG_NONCOPPER_ZONES_PROPERTIES_BASE::DIALOG_NONCOPPER_ZONES_PROPERTIES_BASE(
 	
 	wxString m_OutlineAppearanceCtrlChoices[] = { _("Line"), _("Hatched Outline"), _("Full Hatched") };
 	int m_OutlineAppearanceCtrlNChoices = sizeof( m_OutlineAppearanceCtrlChoices ) / sizeof( wxString );
-	m_OutlineAppearanceCtrl = new wxRadioBox( this, wxID_ANY, _("Outlines Appearence"), wxDefaultPosition, wxDefaultSize, m_OutlineAppearanceCtrlNChoices, m_OutlineAppearanceCtrlChoices, 1, wxRA_SPECIFY_COLS );
+	m_OutlineAppearanceCtrl = new wxRadioBox( this, wxID_ANY, _("Outlines Appearance"), wxDefaultPosition, wxDefaultSize, m_OutlineAppearanceCtrlNChoices, m_OutlineAppearanceCtrlChoices, 1, wxRA_SPECIFY_COLS );
 	m_OutlineAppearanceCtrl->SetSelection( 1 );
 	m_OutilinesBoxOpt->Add( m_OutlineAppearanceCtrl, 0, wxALL|wxEXPAND, 5 );
 	
diff --git a/pcbnew/dialogs/dialog_non_copper_zones_properties_base.fbp b/pcbnew/dialogs/dialog_non_copper_zones_properties_base.fbp
index 20f6296..58ae33a 100644
--- a/pcbnew/dialogs/dialog_non_copper_zones_properties_base.fbp
+++ b/pcbnew/dialogs/dialog_non_copper_zones_properties_base.fbp
@@ -445,7 +445,7 @@
                                                 <property name="gripper">0</property>
                                                 <property name="hidden">0</property>
                                                 <property name="id">wxID_ANY</property>
-                                                <property name="label">Outlines Appearence</property>
+                                                <property name="label">Outlines Appearance</property>
                                                 <property name="majorDimension">1</property>
                                                 <property name="max_size"></property>
                                                 <property name="maximize_button">0</property>
diff --git a/pcbnew/dialogs/wizard_3DShape_Libs_downloader.cpp b/pcbnew/dialogs/wizard_3DShape_Libs_downloader.cpp
index 08e61ff..6506552 100644
--- a/pcbnew/dialogs/wizard_3DShape_Libs_downloader.cpp
+++ b/pcbnew/dialogs/wizard_3DShape_Libs_downloader.cpp
@@ -107,7 +107,7 @@ WIZARD_3DSHAPE_LIBS_DOWNLOADER::WIZARD_3DSHAPE_LIBS_DOWNLOADER( wxWindow* aParen
     // and not fully visible.
     // Forcing deselection does not work, at least on W7 with wxWidgets 3.0.2
     // So (and also because m_textCtrlGithubURL and m_downloadDir are rarely modified
-    // the focus is given to an other widget.
+    // the focus is given to another widget.
     m_hyperlinkGithubKicad->SetFocus();
 
     Connect( wxEVT_RADIOBUTTON, wxCommandEventHandler( WIZARD_3DSHAPE_LIBS_DOWNLOADER::OnSourceCheck ), NULL, this );
diff --git a/pcbnew/drc_clearance_test_functions.cpp b/pcbnew/drc_clearance_test_functions.cpp
index 39d1b9d..4c44035 100644
--- a/pcbnew/drc_clearance_test_functions.cpp
+++ b/pcbnew/drc_clearance_test_functions.cpp
@@ -278,7 +278,7 @@ bool DRC::doTrackDrc( TRACK* aRefSeg, TRACK* aStart, bool testPads )
         {
             D_PAD* pad = m_pcb->GetPad( ii );
 
-            /* No problem if pads are on an other layer,
+            /* No problem if pads are on another layer,
              * But if a drill hole exists	(a pad on a single layer can have a hole!)
              * we must test the hole
              */
@@ -409,7 +409,7 @@ bool DRC::doTrackDrc( TRACK* aRefSeg, TRACK* aStart, bool testPads )
 
         /*	We have changed axis:
          *  the reference segment is Horizontal.
-         *  3 cases : the segment to test can be parallel, perpendicular or have an other direction
+         *  3 cases : the segment to test can be parallel, perpendicular or have another direction
          */
         if( segStartPoint.y == segEndPoint.y ) // parallel segments
         {
diff --git a/pcbnew/onrightclick.cpp b/pcbnew/onrightclick.cpp
index 68c83a3..f673df9 100644
--- a/pcbnew/onrightclick.cpp
+++ b/pcbnew/onrightclick.cpp
@@ -635,7 +635,7 @@ void PCB_EDIT_FRAME::createPopupMenuForTracks( TRACK* Track, wxMenu* PopMenu )
         }
     }
 
-    // Allows switching to an other track/via size when routing
+    // Allows switching to another track/via size when routing
     AddMenuItem( PopMenu, Append_Track_Width_List( GetBoard() ), ID_POPUP_PCB_SELECT_WIDTH,
                  _( "Select Track Width" ), KiBitmap( width_track_xpm ) );
 
diff --git a/pcbnew/pcbnew.cpp b/pcbnew/pcbnew.cpp
index c6421ed..e2b399a 100644
--- a/pcbnew/pcbnew.cpp
+++ b/pcbnew/pcbnew.cpp
@@ -214,7 +214,7 @@ static bool scriptingSetup()
     kipython = fn.GetPath();
 
     // If our python install is existing inside kicad, use it
-    // Note: this is usefull only when an other python version is installed
+    // Note: this is usefull only when another python version is installed
     if( wxDirExists( kipython ) )
     {
         // clear any PYTHONPATH and PYTHONHOME env var definition: the default
diff --git a/pcbnew/plot_board_layers.cpp b/pcbnew/plot_board_layers.cpp
index 34c08fd..6345395 100644
--- a/pcbnew/plot_board_layers.cpp
+++ b/pcbnew/plot_board_layers.cpp
@@ -963,7 +963,7 @@ PLOTTER* StartPlotBoard( BOARD *aBoard, PCB_PLOT_PARAMS *aPlotOpts,
         HPGL_plotter = new HPGL_PLOTTER();
 
         /* HPGL options are a little more convoluted to compute, so
-           they're split in an other function */
+           they're split in another function */
         ConfigureHPGLPenSizes( HPGL_plotter, aPlotOpts );
         plotter = HPGL_plotter;
         break;
diff --git a/pcbnew/ratsnest.cpp b/pcbnew/ratsnest.cpp
index b388085..ee00c22 100644
--- a/pcbnew/ratsnest.cpp
+++ b/pcbnew/ratsnest.cpp
@@ -342,7 +342,7 @@ static int tst_links_between_blocks( NETINFO_ITEM*          aNetinfo,
     int            subratsnest_id, min_id;
     RATSNEST_ITEM* link, * best_link;
 
-    // Search a link from a block to an other block
+    // Search a link from a block to another block
     best_link = NULL;
 
     for( unsigned ii = aNetinfo->m_RatsnestStartIdx; ii < aNetinfo->m_RatsnestEndIdx; ii++ )
@@ -456,7 +456,7 @@ static void tst_links_between_pads( int &      aCurrSubRatsnestId,
  * - First:
  *   Initialise for each pad the subratsnest id to its subnet value
  *   explore the full ratnest (relative to the net) and active a link each time at least one pad of
- *   the given link is not connected to an other pad by a track ( subratsnest = 0)
+ *   the given link is not connected to another pad by a track ( subratsnest = 0)
  *   If the 2 pads linked have both the subratsnest id = 0, a new subratsnest value is created
  * -  Second:
  *   explore the full ratnest (relative to the net) and find a link that links
diff --git a/pcbnew/zones_by_polygon.cpp b/pcbnew/zones_by_polygon.cpp
index 31db133..4e833a0 100644
--- a/pcbnew/zones_by_polygon.cpp
+++ b/pcbnew/zones_by_polygon.cpp
@@ -129,7 +129,7 @@ void PCB_EDIT_FRAME::duplicateZone( wxDC* aDC, ZONE_CONTAINER* aZone )
     {
         DisplayError( this,
             _( "The duplicated zone is on the same layer as the initial zone, which has no sense.\n"
-               "Please, choose an other layer for the new zone") );
+               "Please, choose another layer for the new zone") );
         success = false;
     }
 
@@ -684,7 +684,7 @@ int PCB_EDIT_FRAME::Begin_Zone( wxDC* DC )
             // PCB_EDIT_FRAME::SetCurItem() calls DisplayInfo().
             GetScreen()->SetCurItem( NULL );
             DisplayError( this,
-                          _( "DRC error: this start point is inside or too close an other area" ) );
+                          _( "DRC error: this start point is inside or too close another area" ) );
             return 0;
         }
 
@@ -743,7 +743,7 @@ bool PCB_EDIT_FRAME::End_Zone( wxDC* DC )
         if( g_Drc_On && m_drc->Drc( zone, icorner ) == BAD_DRC )      // we can't validate the closing edge
         {
             DisplayError( this,
-                          _( "DRC error: closing this area creates a DRC error with an other area" ) );
+                          _( "DRC error: closing this area creates a DRC error with another area" ) );
             m_canvas->MoveCursorToCrossHair();
             return false;
         }
diff --git a/pcbnew/zones_functions_for_undo_redo.cpp b/pcbnew/zones_functions_for_undo_redo.cpp
index 65e761f..46eeb95 100644
--- a/pcbnew/zones_functions_for_undo_redo.cpp
+++ b/pcbnew/zones_functions_for_undo_redo.cpp
@@ -184,7 +184,7 @@ int SaveCopyOfZones( PICKED_ITEMS_LIST& aPickList, BOARD* aPcb, int aNetCode, LA
  *          its status becomes UR_DELETED
  *          the aAuxiliaryList corresponding picker is removed (if not found : set an error)
  *  >> if the picked zone was flagged as UR_NEW, and was after deleted ,
- *  perhaps combined with an other zone  (i.e. not found in board list):
+ *  perhaps combined with another zone  (i.e. not found in board list):
  *          the picker is removed
  *          the zone itself if really deleted
  *          the aAuxiliaryList corresponding picker is removed (if not found : set an error)
@@ -262,7 +262,7 @@ void UpdateCopyOfZonesList( PICKED_ITEMS_LIST& aPickList,
                 {
                     DBG( printf(
                         "UpdateCopyOfZonesList(): item not found in aAuxiliaryList,"
-                        "combined with an other zone\n" ) );
+                        "combined with another zone\n" ) );
                 }
                 break;
             }
diff --git a/pcbnew/zones_test_and_combine_areas.cpp b/pcbnew/zones_test_and_combine_areas.cpp
index 9793756..667e72a 100644
--- a/pcbnew/zones_test_and_combine_areas.cpp
+++ b/pcbnew/zones_test_and_combine_areas.cpp
@@ -253,7 +253,7 @@ bool BOARD::TestAreaIntersection( ZONE_CONTAINER* area_ref, ZONE_CONTAINER* area
         }
     }
 
-    // If a contour is inside an other contour, no segments intersects, but the zones
+    // If a contour is inside another contour, no segments intersects, but the zones
     // can be combined if a corner is inside an outline (only one corner is enought)
     for( int ic2 = 0; ic2 < poly2->GetCornersCount(); ic2++ )
     {
